COPY/etc/httpd/conf.d/cfme-https-application.conf

```diff
@@ -5,9 +5,6 @@ KeepAlive on
 
 DocumentRoot /var/www/miq/vmdb/public
 
-# Serve self service directly from public and avoid hitting Rails
-Alias /self_service /var/www/miq/vmdb/public/self_service
-
```

Comment:
Somehow my comment got lost...

@gtanzillo @abellotti Isn't there some symlink on the appliance that can also be removed if this is removed?